setup.py

```diff
@@ -15,9 +15,9 @@
 VERSION = None
 
 REQUIRED = [
-    'requests==2.20.1',
-    'PyJWT==1.6.4',
-    'cryptography==2.6.1',
+    'requests>=2.20.1,==2.*',
+    'PyJWT>=1.6.4,==1.*',
+    'cryptography>=2.6.1,==2.*',
```

Comment:
I know we can't switch to PyJWT 2.* without adjusting our code.
But what about cryptography 3.* ? Are there any breaking changes?

Reply:
I wasn't sure it was compatible so I erred on the safe side. Changed.

Reply:
Actually I don't know either... I was just asking if you knew 😄 
Do you mind reverting back to your previous version?
I'll try to update dependencies when I have setup tests (I am working on it)

Reply:
Haha, I don't really know.

Reverted then :)

The tests would be a great addition!